forecast-admin/forecast/opportunities/fixtures/users.yaml

```diff
@@ -0,0 +1,13 @@
+- fields:
+    is_active: true
+    last_name: ''
+    user_permissions: []
+    first_name: ''
+    is_superuser: true
+    email: ''
+    username: admin
+    password: pbkdf2_sha256$20000$SeBQtWDFH2HW$FPmflJ4hN0yk4uPS47gqMyqEt2WhSWFVxOqUFLRxuLE=
```

Comment:
It isn't strictly necessary. I used it when I was doing local dev because I had to wipe my db enough. Happy to remove it.